### PR TITLE
Added PL Code of Conduct

### PR DESCRIPTION
**Pull request by sidntrivedi012**
@jywarren Please review. Have added the Public Lab Code of Conduct. Solves #105

### CODE REVIEW

File: Code_Of_Conduct.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Public Lab Code of Conduct
+
+_Public Lab, PO Box 426113, Cambridge, MA 02142_
```
Comment (jywarren) [inline Code_Of_Conduct.md]: Hi, can we remove this line, as our address has changed? Let's just let the URL at bottom stand. Thanks!

Comment (sidntrivedi012) [reply Code_Of_Conduct.md]: Sure. Making the changes.

**Comment by sidntrivedi012** [comment]
@jywarren PTAL. Thanks.

**Comment by sidntrivedi012** [comment]
@jywarren PTAL. I think it's ready to be merged. Thanks.

**Comment by sidntrivedi012** [comment]
Or wait, let me close it and resend the PR for both CoC and Contributing Guide from a different branch. I had foolishly sent this PR from the main branch of my fork.

**Comment by jywarren** [comment]
ok, thanks!

**Pull request closed without merging**